css/airspace.css

```diff
@@ -2778,6 +2778,28 @@ font-size:30px;
   top:0.1em;
   color: #fff !important;
 }  
+.join-mat {
+  display:inline-block;
+  padding:0.7em 1.4em;
+  margin:0 0.3em 0.3em 0;
+  border-radius:0.15em;
+  box-sizing: border-box;
+  text-decoration:none;
+  font-family:'Roboto',sans-serif;
+  text-transform:uppercase;
+  font-weight:400;
+  color:#FFFFFF !important;
+  background-color:#00a40e;
+  box-shadow:inset 0 -0.6em 0 -0.35em rgba(0,0,0,0.17);
+  text-align:center;
+  position:relative;
+  transition: all 0.3s cubic-bezier(.25,.8,.25,1);
+  font-size: 1.5em;
+}
+.join-mat:active {
+  top:0.1em;
+  color: #fff !important;
+}  
```

Comment:
Do not change css/airspace.css, as this will hinder our merge of upstream change.  Change css/custom.css instead, or another css file.